tests/muxcable_test.py

```diff
@@ -487,6 +490,51 @@ def test_config_muxcable_disable_loopback(self):
 
         assert result.exit_code == 100
 
+    @mock.patch('sonic_y_cable.y_cable.get_pn_number_and_vendor_name', mock.MagicMock(return_value=(bytearray(b'CACL1X321P2PA1M'), bytearray(b'Credo          '))))
+    @mock.patch('show.muxcable.platform_sfputil', mock.MagicMock(return_value=1))
+    @mock.patch('utilities_common.platform_sfputil_helper.logical_port_name_to_physical_port_list', mock.MagicMock(return_value=[0]))
+    def test_show_muxcable_cableinfo(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(show.cli.commands["muxcable"].commands["cableinfo"],
+                               ["Ethernet0"], obj=db)
+
+        assert result.exit_code == 0
```

Comment:
Suggest also verifying the stdout output. You can use the pytest `capsys` fixture. You can see an example of how I'm using it in [this PR](https://github.com/Azure/sonic-utilities/pull/1416/files#diff-e9018263425d7d7eb008d59325941f6000b66f81f5b2bc411f00a4baacc688a2R61-R73).

Reply:
fixed.

Reply:
I don't see the change. Did a commit get lost?

Reply:
`assert result.output == expected_muxcable_cableinfo_output` on line number 511 checks the output against expected

Reply:
Ah, yes. No need for the capsys fixture because this is using Click. :)